tools/languagelocalizer/languageLocalizer.js

```diff
@@ -128,36 +38,41 @@ const LanguageLocalizer = {
 
   async init() {
     program.version('0.0.1')
+    program.option("-c, --check", `
+      Check only, will fail if user input would be required to generate 
+      the file.
+    `);
 
-    program.option(
-        '-p, --path <mozillavpn>',
-        `config file. Default: ${DEFAULT_MOZILLAVPN}`, DEFAULT_MOZILLAVPN);
     program.action(async () => await this.run(program.opts()));
 
     program.parseAsync(process.argv);
   },
 
   async run(options) {
-    this.execClient(options.path);
-    await this.connectToClient();
-
+    this.options = options;
+    if(options.check){
+      console.log("Running Checks Only");
+    }
     this.wbk = WBK({
       instance: 'https://www.wikidata.org',
       sparqlEndpoint: 'https://query.wikidata.org/sparql',
     })
+    const __filename = fileURLToPath(import.meta.url);
+    const __dirname = path.dirname(__filename);
+    const submodule_path = path.resolve(__dirname, I18N_SUBMODULE_PATH)
+    const dir_entries = fs.readdirSync(submodule_path,{withFileTypes:true});
+    const language_folders = dir_entries
+                              .filter(f => f.isDirectory())
+                              .map(f=>f.name)
+                              .filter(f=>!f.startsWith(".")); // no
 
-    const languages = await this.sendMessageToClient('languages');
-    if (!Array.isArray(languages.value) || languages.value.length === 0) {
-      console.log('No languages! Are you authenticated already?');
-      this.quit();
-    }
+    this.languages = language_folders;
```

Comment:
Small additional changes, instead of pulling the languages from a client built on that branch, we can also directly pull the languages from the checked out submodule.